public-api/src/main/resources/public-api.yaml

```diff
@@ -138,6 +138,25 @@ paths:
           schema:
             $ref: "#/definitions/ConceptListResponse"
 
+  /v1/databrowser/searchConceptsEmptyQuery:
```

Comment:
Seems like we could just have the one endpoint with an optional query parameter?